Examples/Debugging/WhereIsThisPainting/lib/api.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: 'https://collectionapi.metmuseum.org/public/collection/v1/',
+  responseType: 'json',
+});
```

Comment:
I think it's a good pattern to create an instances of Axios and export it. Neat:)

Reply:
Thanks 🙂